school-library/person/persons.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
- [ ] The `is_of_age?` method should be a private method as stated in the requirement. Kindly modify it.